core/tests/puppeteer-acceptance-tests/utilities/common/test-constants.ts

```diff
@@ -234,5 +234,9 @@ export default {
     'Software',
     'Art and Design',
   ],
-  DEFAULT_SPEC_TIMEOUT_MSECS: 300000,
+  DEFAULT_SPEC_TIMEOUT_MSECS: 300000, // The video recording path below is relative to the oppia/ folder's
```

Comment:
Not sure why the comment here is on this line ... should you drop it down a line? It is not related to the timeout and comments should always start on their own line.

Reply:
Fixed.